docs/src/content/docs/victory-primitives.md

```diff
@@ -92,6 +92,7 @@ const TSpan = props => <tspan {...props} />;
 **Props**
 
 - `active` _boolean_ a flag signifying whether the component is active
+- `aria-label` _string or function_ label that will be applied to the rendered path. When this prop is given as a function it will be called with the rest of the props supplied to `Arc`
```

Comment:
These should all be `ariaLabel` instead of `aria-label`, but maybe the first line here should read "a prop that controls the aria-label that will be applied to..."

Reply:
great thank you, left over from the VictoryAccessibleGroup props
have changed all to:
 -`ariaLabel` _string or function_ aria-label that will be applied to the rendered...`